### PR TITLE
Cluster tutorial

### PR DESCRIPTION
**Pull request by mikeb01**
Includes:
- Asciidoc cluster tutorial
- Sample code
- Calls to gradle asciidoctor plugin to render during the build.

### CODE REVIEW

File: aeron-samples/src/docs/asciidoc/ClusterTutorial.adoc [hunk 1]
```diff
@@ -0,0 +1,913 @@
...
+
+You appear to be viewing this tutorial on Github.  The document is much nicer to read if you checkout the Aeron project
+and run:
+
+[source]
+----
+$ ./gradlew asciidoctor
+----
+
+This will create a more nicely formatted file: `aeron-samples/build/asciidoc/html5/ArchiverTutorial.html` with inline
```
Comment (epickrram) [inline aeron-samples/src/docs/asciidoc/ClusterTutorial.adoc]: `ClusterTutorial.html`?

Comment (mikeb01) [reply aeron-samples/src/docs/asciidoc/ClusterTutorial.adoc]: Oops.

File: aeron-samples/scripts/cluster/basic-auction-cluster [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+##
+
+source ./script-common
+
+mkdir -p logs
+
+# tag::start_jvm[]
+function startNode() {
+    ${JAVA_HOME}/bin/java \
+        --add-opens java.base/jdk.internal.ref=ALL-UNNAMED \
```
Comment (vico) [inline aeron-samples/scripts/cluster/basic-auction-cluster]: I think this line should be `${ADD_OPENS} \` . I tried to run the script with JDK8 and failed at this line with `Unrecognized option: --add-opens`

Comment (mikeb01) [reply aeron-samples/scripts/cluster/basic-auction-cluster]: Cheers, yes that sample needs to be aware of the jdk version. Glad the tutorial was useful.

Comment (mikeb01) [reply aeron-samples/scripts/cluster/basic-auction-cluster]: That should be fixed now.

**Pull request merged**